### ISSUE TITLE
prefer-deletion-insertion? option

### ISSUE THREAD

**Issue by federkasten** [chrovis/varity#94]
I would like to add the option `prefer-deletion-insertion?` as well as `prefer-deletion?` and `prefer-insertion?`.

**Comment by totakke** [comment]
You mean coding DNA delins will be adopted instead of repeated sequence if `:prefer-deletion-insertion? true`, right?
If so, I agree on your proposal.

**Comment by federkasten** [comment]
Yes, I want deletion-insertion to be prioritized not only for repeated sequences but also for inversions.
For instance, I'd prefer `c.180_181inv` to be represented as `c.180_181delinsGA`.
